diff -urN thunar-volman-0.2.0/configure.in thunar-volman-0.2.0.new/configure.in
--- thunar-volman-0.2.0/configure.in	2007-12-02 15:58:53.000000000 +0100
+++ thunar-volman-0.2.0.new/configure.in	2007-12-07 20:47:20.000000000 +0100
@@ -75,7 +75,7 @@
 dnl ******************************
 dnl *** Check for i18n support ***
 dnl ******************************
-XDT_I18N([de en_GB eu fi fr hu it lv nb_NO pl pt_BR uk ur zh_TW])
+XDT_I18N([de en_GB eu fi fr hu it lv nb pl pt_BR uk ur zh_TW])
 
 dnl ***********************************
 dnl *** Check for required packages ***
diff -urN thunar-volman-0.2.0/po/LINGUAS thunar-volman-0.2.0.new/po/LINGUAS
--- thunar-volman-0.2.0/po/LINGUAS	2007-12-02 15:58:44.000000000 +0100
+++ thunar-volman-0.2.0.new/po/LINGUAS	2007-12-07 20:47:30.000000000 +0100
@@ -1,2 +1,2 @@
 # set of available languages (in alphabetic order)
-de en_GB eu fi fr hu it lv nb_NO pl pt_BR uk ur zh_TW
+de en_GB eu fi fr hu it lv nb pl pt_BR uk ur zh_TW
